sql-select/twenty-films-of-06.sql

```diff
@@ -0,0 +1,6 @@
+select  "filmId",
```

Comment:
`"filmId"` isn't in the list of fields that should be selected.

---

![image](https://user-images.githubusercontent.com/16143771/144165359-1cf72aec-1182-4270-a2de-dd49a4e761b7.png)